webviewer/webview/sys_windows.go

```diff
@@ -48,10 +48,8 @@ func init() {
 	}
 }
 
-var (
-	// referenceHolder prevents GC from releasing the COM object.
-	referenceHolder = make(map[unsafe.Pointer]struct{}, 64)
-)
+// referenceHolder prevents GC from releasing the COM object.
+var referenceHolder = make(map[unsafe.Pointer]struct{}, 64)
```

Comment:
Almost all declarations in that file is using `var ( ... )` or `type ( ... )`. I think don't make sense to change it.